pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/prerequisites-airgapped/index.md

```diff
@@ -0,0 +1,167 @@
+    
+    ```bash
+    export DKP_VERSION=v2.2.0
+    ```
+
+2.  Download the bootstrap docker image on a machine that has access to this artifact.
+
+    ```docker
+    curl -O https://downloads.d2iq.com/dkp/$DKP_VERSION/konvoy-bootstrap_$DKP_VERSION.tar
+    ```
+
+3.  Load the bootstrap docker image on your bastion machine.
+
+    ```docker
+    docker load -i konvoy-bootstrap_$DKP_VERSION.tar
+    ```
+
+## Copy air-gapped artifacts onto cluster hosts 
+
+Using the [Konvoy Image Builder](../../../image-builder), you can copy the required artifacts onto your cluster hosts.
+
+1.  Create the directories where you will place the air-gapped bundles:
+
+    ```
+    mkdir artifacts
+    mkdir artifacts/images
+    ```
+
+1.  Define an environment variable for the Kubernetes version that corresponds with Konvoy release you are installing. You can find the correct Kubernetes version by checking the release notes for the release you are installing.
+
+    ```bash
+    export VERSION=1.22.8
+    ```
+
+1.  Set an environment variable for the AMI's OS you are will be using.
+    The OS packages bundles will contain the RPMs for Containerd, Kubernetes and all of their dependencies required to install these packages without access to any external RPM repositories.
+    The available options are:
+
+    * `centos_7_x86_64`
+    * `centos_7_x86_64_fips`
+    * `redhat_7_x86_64`
+    * `redhat_7_x86_64_fips`
+    * `redhat_8_x86_64`
+    * `redhat_8_x86_64_fips`
+
+    ```
+    export BUNDLE_OS=centos_7_x86_64
+    ```
+
+1.  Download the OS packages bundle:
+
+    ```
+    curl --output artifacts/"$VERSION"_"$BUNDLE_OS".tar.gz -O https://downloads.d2iq.com/dkp/airgapped/os-packages/"$VERSION"_"$BUNDLE_OS".tar.gz
+    ```
+
+1.  Download the Kubernetes images bundle. This bundle includes the necessary images for `kubeadm` to bootstrap a Kubernete `Node`.
+
+    The available options for each Kubernetes version are:
+
+    * `<version>_images.tar.gz`
+    * `<version>_images_fips.tar.gz`
+
+    ```
+    curl --output artifacts/images/"$VERSION"_images.tar.gz -O https://downloads.d2iq.com/dkp/airgapped/kubernetes-images/"$VERSION"_images.tar.gz
+    ```
+
+1.  Download the PIP packages. This bundle includes a few packages required by DKP to boostrap machines.
+
+    ```
+    curl --output artifacts/pip-packages.tar.gz -O https://downloads.d2iq.com/dkp/airgapped/pip-packages/pip-packages.tar.gz
+    ```
+
+1.  Export these environment variables:
+
+    ```bash
+    export CONTROL_PLANE_1_ADDRESS="<control-plane-address-1>"
+    export CONTROL_PLANE_2_ADDRESS="<control-plane-address-2>"
+    export CONTROL_PLANE_3_ADDRESS="<control-plane-address-3>"
+    export WORKER_1_ADDRESS="<worker-address-1>"
+    export WORKER_2_ADDRESS="<worker-address-2>"
+    export WORKER_3_ADDRESS="<worker-address-3>"
+    export WORKER_4_ADDRESS="<worker-address-4>"
+    export SSH_USER="<ssh-user>"
+    export SSH_PRIVATE_KEY_SECRET_NAME="$CLUSTER_NAME-ssh-key"
```

Comment:
Its a little tricky because of how `konvoy-image` works running in a docker container and with volume bounds. I would prefer to keep it this way (which already follows the existing steps in the create cluster page).